pkg/utils/cluster.go

```diff
@@ -53,15 +55,15 @@ func IsSingleNodeCluster(c client.Client) (bool, error) {
 // On kubernetes, it is determined by which node the sriov operator is scheduled on. If operator
 // pod is schedule on worker node, it is considered as external control plane.
 func IsExternalControlPlaneCluster(c client.Client) (bool, error) {
-	if os.Getenv("CLUSTER_TYPE") == ClusterTypeOpenshift {
+	if os.Getenv("CLUSTER_TYPE") == consts.ClusterTypeOpenshift {
```

Comment:
As we created `vars.ClusterType`, we can use that instead of `os.Getenv("CLUSTER_TYPE")`

Same on L40 and L66

Reply:
I have it on the next PR https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/555/commits/f464e8acbe1151cb684ac106b96f745ae85f1c18

this one is already pretty big :P

Reply:
fine with that!